sandbox/server/walker/walk_creator.py

```diff
@@ -0,0 +1,133 @@
+
```

Comment:
Can you update this file with the new `walk_creator` from the main branch? https://github.com/CarperAI/ELM/blob/main/walker/walk_creator.py